<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="ViewMembers">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>811</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Menu">
     <attribute name="title">
      <string>Menu</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="Items">
     <attribute name="title">
      <string>Items</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidgetItem">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>781</width>
        <height>461</height>
       </rect>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Amount</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>490</y>
        <width>781</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButtonAdd">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButtonEdit">
         <property name="text">
          <string>Edit</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButtonRemove">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="pushButton_ItemReport">
         <property name="text">
          <string>View Reports</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="members">
     <attribute name="title">
      <string>Members</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidgetMembers">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>761</width>
        <height>441</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::NoPen</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Expiration Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Membership Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Total Spent</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>500</y>
        <width>771</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_MemberAdd">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_MemberEdit">
         <property name="text">
          <string>Edit</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_MemberRemove">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>View Report</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Reports</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>791</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>761</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Click on the type of report you want to view:</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>781</width>
        <height>121</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QPushButton" name="SalesReport_Button">
         <property name="text">
          <string>Sales Report</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="TotalPurchases_Button">
         <property name="text">
          <string>Total Purchases</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="ItemsSoldQuantity_Button">
         <property name="text">
          <string>Items Sold - Quantity</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="Rebates_Button">
         <property name="text">
          <string>Rebates Due</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="Expire_Button">
         <property name="text">
          <string>Memberships to Expire</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="ItemsSoldName_Button">
         <property name="text">
          <string>Items Sold - Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="MemberPurchases_Button">
         <property name="text">
          <string>Member Purchases</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="Convert_Button">
         <property name="text">
          <string>Members Who Should Convert</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>885</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
